themes/classic/templates/checkout/order-confirmation.tpl

```diff
@@ -10,12 +10,82 @@
   </section>
 
   <section id="content-hook_order_confirmation">
+    <h3>{l s='Your order is confirmed'}</h3>
+    {if $url_to_invoice !== ''}
+    <div>
+        {l s='An email has been sent to your mail address'}{* user email address *}
+        {l s='You can also'}<a href="{$url_to_invoice}">{l s='Download your invoice'}</a>
+    </div>
+    {/if}
     {$HOOK_ORDER_CONFIRMATION}
```

Comment:
no, no, not `|escape:'htmlall':'utf-8'` just `nofilter` because variables are escaped by default and for example bankwire will return HTML so we need to `nofilter` this variable in my opinion